src/View/Helper/WebComponentsHelper.php

```diff
@@ -0,0 +1,97 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2020 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+namespace BEdita\WebTools\View\Helper;
+
+use Cake\View\Helper\HtmlHelper as CakeHtmlHelper;
+
+/**
+ * Helper to handle Web Components initialization with properties.
+ */
+class WebComponentsHelper extends CakeHtmlHelper
```

Comment:
There is a reason to extend `HtmlHelper`? Since we don't need to override any `HtmlHelper`method I would prefer to extend `Cake\View\Helper` and use `HtmlHelper` inside `WebComponentsHelper`

```php
public $helpers = ['Html'];
```